async/actorpromise.md

```diff
@@ -0,0 +1,127 @@
+# Interrogating Actors with Promises
+
+## Problem
+Pony gives us an excellent abstraction for actors. We can define fields within those actors to maintain state and rely on the single-threaded nature of inbound message processing to ensure safe access to those fields. A problem arises when one actor wants to access the internal _state_ of another actor.
+
+Let's say that we've built an event-sourced system that is dealing with financial transactions. We have an actor called `AccountAggregate` that is maintaining an _internal_ balance. This balance is determined by receiving events in the form of transactions (withdrawals, deposits, transfers, adjustments, etc). This sequence of events is processed to compute the balance for any given account.
```

Comment:
Event sourcing never comes into play in this that I can see. I think it makes sense to drop mention of it as not everyone knows what it is and is liable to lead to confusion.